.rubocop.yml

```diff
@@ -20,3 +20,7 @@ Rails:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+Metrics/MethodLength:
+  Exclude:
+    - 'spec/requests/*'
```

Comment:
Yeah, me too. I know we are going to get a bunch of violations regarding this rule in the request specs by their very nature. Happy to explicitly add each individual request spec, rather than glob them all, if you prefer that?